osgp/protocol-adapter-dlms/osgp-jasper-interface/src/main/java/org/opensmartgridplatform/adapter/protocol/jasper/config/JasperWirelessConfig.java

```diff
@@ -97,20 +94,24 @@ public SaajSoapMessageFactory messageFactory() throws OsgpJasperException {
     }
 
     @Bean
-    public Wss4jSecurityInterceptor wss4jSecurityInterceptorClient() {
+    Wss4jSecurityInterceptor wss4jSecurityInterceptorClient() {
         final Wss4jSecurityInterceptor wss4jSecurityInterceptor = new Wss4jSecurityInterceptor();
         wss4jSecurityInterceptor.setSecurementActions("UsernameToken");
         return wss4jSecurityInterceptor;
     }
 
     @Bean
-    public WebServiceTemplate webServiceTemplate() throws OsgpJasperException {
+    public WebServiceTemplate jasperWebServiceTemplate() throws OsgpJasperException {
```

Comment:
renamed this so it is clearer where the bean comes from (when searching by name).

Reply:
fine